cdap-ui/app/directives/datetime-range/datetime-range.js

```diff
@@ -25,11 +25,17 @@ function RangeController ($scope) {
   vm.openStartTime = () => {
     vm.startTimeOpen = true;
     vm.endTimeOpen = false;
+
+    document.body.removeEventListener('click', vm.close, false);
```

Comment:
May be instead of repeatedly removing and adding event listeners we could just do it once and remove them in the `$destroy` method?